radiomanager/radiomanager/__init__.py

```diff
@@ -34,14 +23,47 @@ class TESTING_CONFIG:
     SQLALCHEMY_DATABASE_URI = "sqlite://"
 
 
-def create_app(config_obj=PRODUCTION_CONFIG):
+def create_app(config_obj=TESTING_CONFIG):
```

Comment:
This disables persistence IIRC. This should be reverted.

Reply:
changed